Extension/package.json

```diff
@@ -955,6 +955,26 @@
                     "default": {}
                   }
                 }
+              },
+              "symbolLoadInfo": {
```

Comment:
Just an FYI, the CMake Tools extension exposes a lot of the debugger configuration properties via its settings to allow for them to debug stuff without launch.json.  It might be worth it to check and see how out of sync the CMake extension is with cpptools.

Reply:
Is it just the stuff under [debugConfig](https://github.com/microsoft/vscode-cmake-tools/blob/2670472e461e5f6e2b0022c0531e11a349809de8/package.json#L597)?

Reply:
We should generate a schema from our launch.json so that it is imported.

Reply:
Does it have loc? Right now it will inject %c_cpp.debugger.....% into the description of the config.

Reply:
I have a PR out to add loc to cmake-tools.  https://github.com/microsoft/vscode-cmake-tools/pull/757